src/test/java/org/galatea/starter/entrypoint/IexRestControllerTest.java

```diff
@@ -81,4 +81,30 @@ public void testGetLastTradedPriceEmpty() throws Exception {
         .andExpect(jsonPath("$", is(Collections.emptyList())))
         .andReturn();
   }
+
+  @Test
+  public void testHistoricalPrices() throws  Exception {
+    MvcResult result = this.mvc.perform(
+        org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+            .get("/iex/historicalPrices?symbol=FB&range=1d")
+            .accept(MediaType.APPLICATION_JSON_VALUE))
+        .andExpect(status().isOk())
+        .andExpect(jsonPath("$[0].high").value(new BigDecimal("340.81")))
+        .andExpect(jsonPath("$[1].low").value(new BigDecimal("339.12")))
+        .andExpect(jsonPath("$[2].open").value(new BigDecimal("339.84")))
+        .andExpect(jsonPath("$[3].close").value(new BigDecimal("340.07")))
+        .andExpect(jsonPath("$[5].volume").value(new BigDecimal(5515)))
+        .andReturn();
+  }
+
```

Comment:
Need tests for each case:
* No Date
* No Symbol
* No Range
* All Three
* each combination of two
* etc..